app/helpers/template_helper.py

```diff
@@ -38,6 +33,10 @@ def with_metadata_context(func):
     @wraps(wraps)
     def metadata_context_wrapper(*args, **kwargs):
         metadata = get_metadata(current_user)
+
+        if not metadata:
+            raise Unauthorized
```

Comment:
Is this not handled elsewhere?
can you get this far without having metadata?

Reply:
I added this in when going to feedback page without a session threw a 500, but it seems to magically work now. I'll remove.